packages/frontpage/lib/data/atproto/comment.ts

```diff
@@ -30,13 +30,19 @@ export const CommentRecord = z.object({
 
 export type Comment = z.infer<typeof CommentRecord>;
 
-type CommentInput = {
+export type CommentInput = {
   parent?: { cid: string; rkey: string; authorDid: DID };
   post: { cid: string; rkey: string; authorDid: DID };
   content: string;
+  rkey: string;
 };
 
-export async function createComment({ parent, post, content }: CommentInput) {
+export async function createComment({
+  parent,
+  post,
+  content,
+  rkey,
+}: CommentInput) {
   // Collapse newlines into a single \n\n and trim whitespace
   const sanitizedContent = content.replace(/\n\n+/g, "\n\n").trim();
```

Comment:
We need to perform sanitisation at the API layer so it's shared across atproto and db.